enzyme/Enzyme/CApi.cpp

```diff
@@ -391,17 +391,18 @@ LLVMValueRef EnzymeCreateForwardDiff(
   }
   return wrap(eunwrap(Logic).CreateForwardDiff(
       cast<Function>(unwrap(todiff)), (DIFFE_TYPE)retType, nconstant_args,
-      eunwrap(TA).TLI, eunwrap(TA), returnValue, (DerivativeMode)mode,
+      eunwrap(TA).TLI, eunwrap(TA), returnValue, (DerivativeMode)mode, width,
       unwrap(additionalArg), eunwrap(typeInfo, cast<Function>(unwrap(todiff))),
       uncacheable_args, PostOpt));
 }
 LLVMValueRef EnzymeCreatePrimalAndGradient(
     EnzymeLogicRef Logic, LLVMValueRef todiff, CDIFFE_TYPE retType,
     CDIFFE_TYPE *constant_args, size_t constant_args_size,
     EnzymeTypeAnalysisRef TA, uint8_t returnValue, uint8_t dretUsed,
-    CDerivativeMode mode, LLVMTypeRef additionalArg, CFnTypeInfo typeInfo,
-    uint8_t *_uncacheable_args, size_t uncacheable_args_size,
-    EnzymeAugmentedReturnPtr augmented, uint8_t AtomicAdd, uint8_t PostOpt) {
+    CDerivativeMode mode, unsigned width, LLVMTypeRef additionalArg,
```

Comment:
Please open associate PR to Enzyme.jl on such API changes.,

Reply:
https://github.com/wsmoses/Enzyme.jl/pull/179
https://github.com/rust-ml/oxide-enzyme/pull/12